packages/gas-fee-controller/src/GasFeeController.ts

```diff
@@ -346,9 +346,6 @@ export class GasFeeController extends BaseControllerV2<
         'NetworkController:getProviderConfig',
       );
       this.currentChainId = providerConfig.chainId;
-      this.ethQuery = this.messagingSystem.call(
```

Comment:
In retrospect, my assumption here wasn't entirely true; in practice this likely was set upon construction. Just not in the tests. So this is breaking after all. We'll need to fix this before the next release.

Reply:
Maybe the simplest fix would be to update `getEthQuery` to return `EthQuery | undefined`, and restore this line.

Reply:
Okay, fixed via https://github.com/MetaMask/core/pull/1284.